OSPSuite.Presentation.Importer/Presenters/IColumnMappingPresenter.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using OSPSuite.Core.Importer;
+using OSPSuite.Presentation.Importer.Core;
+using OSPSuite.Presentation.Importer.Core.DataFormat;
+using OSPSuite.Presentation.Importer.Views;
+using OSPSuite.Presentation.Presenters;
+
+namespace OSPSuite.Presentation.Importer.Presenters
+{
+   /// <summary>
+   ///    Event arguments for OnMissingMapping event.
+   /// </summary>
+   public class MissingMappingEventArgs : EventArgs
+   {
+      /// <summary>
+      ///    FileName describing what is missed.
+      /// </summary>
+      public string Message { get; set; }
+   }
+
+   /// <summary>
+   ///    Handler for OnMissingMapping event.
+   /// </summary>
+   public delegate void MissingMappingHandler(object sender, MissingMappingEventArgs e);
+
+   public delegate void MappingCompletedHandler(object sender);
+
+   public class ColumnMappingOption
+   {
+      public string Label { get; set; }
+      public int IconIndex { get; set; }
+      public string Description { get; set; }
+
+      public enum DescriptionType
+      {
+         Ignored,
+         GroupBy,
+         MetaData,
+         Mapping,
+         AddGroupBy
+      }
+   }
+
+   public class ToolTipDescription
+   {
+      public string Title { get; set; }
+      public string Description { get; set; }
+   }
+
+   public class ButtonsConfiguration
+   {
+      public bool ShowButtons { get; set; }
+      public bool UnitActive { get; set; }
+   }
+
+   public interface IColumnMappingPresenter : IPresenter<IColumnMappingControl>
+   {
+      void SetSettings(
```

Comment:
what are settings?

Reply:
We are not sure what is not clear here. Let's also discuss this on our meeting.

Reply:
Settings are for me things that the user can set (Colors, fonts, etc.)
I do not understand what settings means in this context.

Reply:
metaDataCategories and columnInfos are settings for the mappingPresenter. metaDataCategories are comming from the system settings (MoBi or PKSim) and the columnInfos as well. These we could change on the stored settings for the application.